api/host.go

```diff
@@ -148,18 +148,18 @@ func (opts HostsForScanningOptions) Apply(values url.Values) {
 
 type (
 	Host struct {
-		KnownSince       time.Time            `json:"knownSince"`
-		LastAnnouncement time.Time            `json:"lastAnnouncement"`
-		PublicKey        types.PublicKey      `json:"publicKey"`
-		NetAddress       string               `json:"netAddress"`
-		PriceTable       HostPriceTable       `json:"priceTable"`
-		Settings         rhpv2.HostSettings   `json:"settings"`
-		Interactions     HostInteractions     `json:"interactions"`
-		Scanned          bool                 `json:"scanned"`
-		Blocked          bool                 `json:"blocked"`
-		Checks           map[string]HostCheck `json:"checks"`
-		StoredData       uint64               `json:"storedData"`
-		Subnets          []string             `json:"subnets"`
```

Comment:
Unfortunately we need to keep `Subnets` here and in `HostScan`. Even if we don't use it ourselves we need to populate it. But you can add removing that field to https://github.com/SiaFoundation/renterd/issues/1136.

Otherwise we'd break the API and we aren't ready to release a new major version yet.